pages/card-sorting.md

```diff
@@ -6,38 +6,38 @@ title: Card sorting
 
 ## What it is
 
-A categorization exercise in which participants divide concepts into different groups based on their understanding of those concepts. 
+A categorization exercise in which participants divide concepts into different groups based on their understanding of those concepts.
 
 ## Reasons to use it
 
 To create content structures that are user-informed and intuitive.
 
-
 ## Time to conduct
 
 15–30 minutes per user
 
-
 ## How to do it
 
 There are two types of card sorting: open and closed. Most card sorts are performed with one user at a time, but you can also do the exercise with groups of two to three people.
 
-
 ## Open card sort
 
 1. Give users a collection of content represented on cards.
-2. Ask users to separate the cards into whatever categories make sense to them. 
-3. Ask users to label those categories. 
-4. Ask users to tell you why they grouped the cards and labeled the categories as they did.
 
+2. Ask users to separate the cards into whatever categories make sense to them.
+
+3. Ask users to label those categories.
+
+4. Ask users to tell you why they grouped the cards and labeled the categories as they did.
 
 ## Closed card sort
 
 1. Give users a collection of content represented on cards.
+
 2. Ask the user to parse it into a list of categories you have predefined.
-3. Ask users to tell you why they assigned cards to the categories they did.
 
+3. Ask users to tell you why they assigned cards to the categories they did.
 
 ## Applied in government research
 
--  No PRA implications. The PRA explicitly exempts direct observation and non-standardized conversation 5 CFR 1320.5.(h)3. It also explicitly excludes tests of knowledge or aptitude 5 CFR 1320.5(h)7, which is essentially what a card sort tests (though in our case, a poor result is our fault).
+-  No PRA implications. The PRA explicitly exempts direct observation and non-standardized conversation 5 CFR 1320.5(h)3. It also explicitly excludes tests of knowledge or aptitude 5 CFR 1320.5(h)7, which is essentially what a card sort tests (though in our case, a poor result is our fault).
```

Comment:
Think there needs to be a comma before 5 CFR 1320.5(h)7:
... knowledge or aptitude, 5 CFR 1320.5(h)7, which is essentially...